content/docs/reference/technical-preview-free-tier.md

```diff
@@ -5,49 +5,27 @@ enableTableOfContents: true
 
 The Neon cloud service is available for free during the Technical Preview. For information about how the Technical Preview fits within Neon's release plans, please refer to the [Neon roadmap](/docs/reference/roadmap).
 
-Technical Preview Free Tier users can create up to three projects in Neon. Projects created using the Technical Preview Free Tier are subject to the following limits:
+The Technical Preview Free Tier has the following limits:
 
-- Project data size is limited to 10GB
-- The Point in Time Reset (PITR) window is limited to 7 days of _reasonable usage_
-- The compute node is limited to 1 vCPU and 256MB of RAM
+- 1 Neon project
+- 10 branches
+- 3 GB of data per branch
+- Up to 3 compute nodes (endpoints), one per branch
+- 1 vCPU and 256MB of RAM per compute node
```

Comment:
@kelvich , could you verify this?

Reply:
Changed to 512MB as per this comment in Slack: https://neondb.slack.com/archives/C033A2WE6BZ/p1669840130644329?thread_ts=1669378336.698969&cid=C033A2WE6BZ